src/server/ApiRoutes/ApiRoutes.js

```diff
@@ -32,7 +32,10 @@ router
 
 router
   .route('/hold/request/:bibId-:itemId')
-  .get(Hold.newHoldRequestServer)
+  .get(Hold.newHoldRequestServer);
```

Comment:
Should we get rid of `route('/hold/request/:bibId-:itemId')` as all the item should have `itemSource `